DEVELOPING.rst

```diff
@@ -267,9 +267,9 @@ Here is example:
 
    $ git checkout master
    $ git checkout -b improve-event-display-icons
-   $ emacsclient some_file.py
```

Comment:
Oh wow 👀

Reply:
Hehe, it's just a documentation :) Changing to `$EDITOR` will make it copy-paste-friendly.